docs/optimization_guide/nncf/ptq/code/ptq_openvino.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+#! [dataset]
+import nncf
+import torch
+
+calibration_loader = torch.utils.data.DataLoader(...)
+
+def transform_fn(data_item):
+    images, _ = data_item
+    return images.numpy()
```

Comment:
return images

Reply:
Numpy is the source format for OpenVINO. Even so, torch.Tensor is acceptable, I would like to highlight it.